mayastor/tests/replica_timeout.rs

```diff
@@ -78,25 +86,38 @@ async fn replica_stop_cont() {
         println!("waiting for the container to be fully suspended... {}/5", i);
     }
 
-    mayastor.send(async {
-        // we do not determine if the IO completed with an error or not just
-        // that it completes.
-        let _ = dbg!(bdev_io::read_some(NXNAME, 0, 0xff).await);
-        let _ = dbg!(bdev_io::read_some(NXNAME, 0, 0xff).await);
-    });
+    // initiate the read and leave it in the background to time out
+    let nxuri =
+        format!("nvmf://127.0.0.1:8420/nqn.2019-05.io.openebs:{}", NXNAME);
+    Command::new("../target/debug/initiator")
+        .args(&[&nxuri, "read", "/tmp/tmpread"])
+        .stdout(Stdio::piped())
+        .spawn()
+        .expect("should send read from initiator");
 
     println!("IO submitted unfreezing container...");
 
-    for i in 1 .. 6 {
+    // KATO is 5s, wait at least that long
+    let n = 8;
+    for i in 1 .. n + 1 {
```

Comment:
You can do the following instead:
`for i in 1..=  n {`

Reply:
Thanks, fixed. I also added 2s to the delay, 8s is actually right on the edge and I forgot to commit that change.